source/index.html.md

```diff
@@ -5294,6 +5298,410 @@ Payment Requests where you're the creditor and collecting funds from the debtor.
+
+$request->setRequestUrl('https://api.sandbox.split.cash/payment_requests/receivables');
+$request->setRequestMethod('GET');
+$request->setHeaders(array(
+  'authorization' => 'Bearer {access-token}',
+  'accept' => 'application/json'
+));
+
+$client->enqueue($request)->send();
+$response = $client->getResponse();
+
+echo $response->getBody();
+```
+
+```go
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"io/ioutil"
+)
+
+func main() {
+
+	url := "https://api.sandbox.split.cash/payment_requests/receivables"
+
+	req, _ := http.NewRequest("GET", url, nil)
+
+	req.Header.Add("accept", "application/json")
+	req.Header.Add("authorization", "Bearer {access-token}")
+
+	res, _ := http.DefaultClient.Do(req)
+
+	defer res.Body.Close()
+	body, _ := ioutil.ReadAll(res.Body)
+
+	fmt.Println(res)
+	fmt.Println(string(body))
+
+}
+```
+
+`GET /payment_requests/receivables`
+
+Payment Requests where the debtor is sending you funds ([Receivable Contacts](/#add-a-receivable-contact)). This endpoint exposes all received payments.
+
+<h3 id="List-Payment-Requests:-Receivables-parameters" class="parameters">Parameters</h3>
+
+|Parameter|In|Type|Required|Description|
+|---|---|---|---|---|
+|page|query|string|false|Page of results to return, single value, exact match|
+|per_page|query|string|false|Number of results per page, single value, exact match|
+
+> Example responses
+
+> 200 Response
+
+```json
+{
+  "data": [
+    {
+      "ref": "PR.2t65",
+      "initiator_id": "ca7bc5b3-e47f-4153-96fb-bbe326b42772",
+      "your_bank_account_id": "9c70871d-8e36-4c3e-8a9c-c0ee20e7c679",
+      "authoriser_id": "de86472c-c027-4735-a6a7-234366a27fc7",
+      "authoriser_contact_id": "fb6a9252-3818-44dc-b5aa-2195391a746f",
+      "contact_initiated": true,
+      "schedule_ref": null,
+      "status": "approved",
+      "status_reason": null,
+      "matures_at": "2021-05-12T13:43:12Z",
+      "responded_at": "2021-05-12T13:43:12Z",
+      "created_at": "2021-05-12T13:43:12Z",
+      "credit_ref": "C.77b1",
+      "payout": {
+        "amount": 50000,
+        "description": "Deposit to my Trading account",
+        "matures_at": "2021-05-12T13:43:12Z"
+      }
+    },
+    {
+      "ref": "PR.1n644",
+      "initiator_id": "ca7bc5b3-e47f-4153-96fb-bbe326b42772",
+      "your_bank_account_id": "9c70871d-8e36-4c3e-8a9c-c0ee20e7c679",
+      "authoriser_id": "de86472c-c027-4735-a6a7-234366a27fc7",
+      "authoriser_contact_id": "fb6a9252-3818-44dc-b5aa-2195391a746f",
+      "contact_initiated": true,
+      "schedule_ref": null,
+      "status": "approved",
+      "status_reason": null,
+      "matures_at": "2021-06-01T04:34:50Z",
+      "responded_at": null,
+      "created_at": "2021-06-01T04:34:56Z",
+      "credit_ref": "c.54r3",
+      "payout": {
+        "amount": 5000,
+        "description": "Punting account top-up",
+        "matures_at": "2021-06-01T04:34:56Z"
+      }
+    }
+  ]
+}
+```
+
+<h3 id="List Payment Requests: Receivables-responses">Responses</h3>
```

Comment:
In theory, according to HTML5 specification (which is more permissive than previous ones), this id value is invalid: it shouldn't contain whitespace. In practice it might not be a problem. 🤷 

> When specified on HTML elements, the id attribute value must be unique amongst all the IDs in the element's tree and must contain at least one character. The value must not contain any ASCII whitespace.

https://html.spec.whatwg.org/multipage/dom.html#the-id-attribute